.github/dependabot.yml

```diff
@@ -5,4 +5,6 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
-    open-pull-requests-limit: 0  
+    open-pull-requests-limit: 0
```

Comment:
please read the document for every setting in this file
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#open-pull-requests-limit
`open-pull-requests-limit: 0` disabled the "gomod" updates

Reply:
got it, removed